wiz-admission-controller/templates/deployment.yaml

```diff
@@ -10,10 +10,11 @@ spec:
       {{- include "wiz-admission-controller.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
       annotations:
-        {{- toYaml . | nindent 8 }}
-      {{- end }}
+        rollme: {{ randAlphaNum 5 | quote }}
```

Comment:
why do you want this?  you want to redeploy on every push?
You want to redeploy on every configmap change?

Reply:
https://renehernandez.io/notes/rolling-pods-config-changes/